docs/getting-started-guides/docker-multinode/master.sh

```diff
@@ -41,10 +42,10 @@ fi
 
 # Make sure master ip is properly set
 if [ -z ${MASTER_IP} ]; then
-    echo "Please export MASTER_IP in your env"
-    exit 1
+    MASTER_IP=$(hostname -I | awk '{print $1}')
```

Comment:
I don't understand what you mean.
This file will run on the master node and spin up all containers.
We use the ip on master only for `etcd` arguments
It's another thing when deploying workers, they have to specify the master ip